travis/script/functions.sh

```diff
@@ -2,8 +2,9 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 source $SCRIPT_DIR/travis_functions.sh
 source $SCRIPT_DIR/predicate_functions.sh
 
-# idea taken from: http://blog.headius.com/2010/03/jruby-startup-time-tips.html
-export JRUBY_OPTS="${JRUBY_OPTS} -X-C" # disable JIT since these processes are so short lived
+# If JRUBY_OPTS isn't set, use these.
+# see http://docs.travis-ci.com/user/ci-environment/
+export JRUBY_OPTS=${JRUBY_OPTS:---server -Xcompile.invokedynamic=false"}
```

Comment:
Thanks @JonRowe That is deeply embarrassing.  Reminds me of the calculus exam I near got 100% on, but, on one of the questions, after solving some integrals, summed 1 and -1 to get 2, or something ridiculous like that.